docs/reference/search/knn-search.asciidoc

```diff
@@ -100,6 +100,30 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=routing]
 `knn`::
 (Required, object) 
 include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=knn]
++
+.Properties of `knn` object
+[%collapsible%open]
+====
+`field`::
+(Required, string) 
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=knn-field]
+
+`filter`::
```

Comment:
For the old and now deprecated _knn_search API the `filter` is at the same level as the `knn` object and should be lifted out of here. 

https://www.elastic.co/guide/en/elasticsearch/reference/8.4/knn-search-api.html#knn-search-api-request-body

For knn as part of _search and _semantic_search `filter` is part of `knn` which is correct. 

https://www.elastic.co/guide/en/elasticsearch/reference/8.4/search-search.html#search-api-knn

So we just need to change it here for _knn_search

Reply:
Fixed in [1d63e86](https://github.com/elastic/elasticsearch/pull/91503/commits/1d63e86c31160f6a5355f23a19301e07a42c44c8).